src/sentry/api/endpoints/project_group_index.py

```diff
@@ -894,10 +892,19 @@ def delete(self, request, project):
                 )
             )
         else:
-            # missing any kind of filter
-            return Response(
-                {"detail": "You must specify a list of IDs for this operation"}, status=400
-            )
+            try:
+                # bulk mutations are limited to 1000 items
+                # TODO(dcramer): it'd be nice to support more than this, but its
+                # a bit too complicated right now
+                cursor_result, _ = self._search(request, project, {
+                    'limit': 1000,
+                    'paginator_options': {'max_limit': 1000},
+                })
+            except ValidationError as exc:
+                return Response({'detail': six.text_type(exc)}, status=400)
+
+            group_list = list(cursor_result)
+            group_ids = [g.id for g in group_list]
```

Comment:
do you actually use these `group_ids` anywhere?

Reply:
You're right, it's not - I'll remove it.